server/routes/request.ts

```diff
@@ -221,11 +221,35 @@ requestRoutes.post(
 
       next({ status: 500, message: 'Invalid media type' });
     } catch (e) {
-      next({ message: e.message, status: 500 });
+      next({ status: 500, message: e.message });
     }
   }
 );
 
+requestRoutes.get('/count', async (_req, res, next) => {
+  const requestRepository = getRepository(MediaRequest);
+
+  try {
+    const pendingCount = await requestRepository.count({
+      status: MediaRequestStatus.PENDING,
+    });
+    const approvedCount = await requestRepository.count({
+      status: MediaRequestStatus.APPROVED,
+    });
+    const availableCount = await requestRepository.count({
+      status: MediaRequestStatus.AVAILABLE,
+    });
```

Comment:
This won't give you what you want. We don't actually use the request status of available for anything. You will need to do a manual query for requests that have a media relation thats available.

Reply:
Ah, maybe I should just remove this.  I was trying to match the Ombi API response, but I don't feel like the available request count is particularly useful anyway.

Reply:
You can also use the `results` from the pageinfo on the normal request list to get you this sorta data. But as of right now, that endpoint as the same problem with `available` items.

If you want to get the # of available items, regardless of whether or not there was a request, you can always hit the `/media` endpoint.

Reply:
I honestly wasn't going to use the available count anyway, and when you think about it, it's just an arbitrary number that is between 0 and the number of items in your Plex library.  Especially for those of us who used other request systems before.  I don't feel like the available count is particularly useful so I removed it in 469c31a.  Thanks for your quick responses & feedback!